erlang/README.md

```diff
@@ -7,40 +7,42 @@ The applications in this directory make use of the [AtomVM Rebar3 Plugin](https:
 
 Some example programs can only run on specific platform.  The following table summarizes the platforms on which the Erlang example programs can be run.
 
-| Example Program | esp32 | stm32 | generic_unix |
-|-----------------|-------|-------|--------------|
-| arepl_example   | ✅ | ❌ | ✅ |
-| blinky          | ✅ | ✅ | ❌ |
-| deep_sleep      | ✅ | ❌ | ❌ |
-| esp_nvs         | ✅ | ❌ | ❌ |
-| gpio_interrupt  | ✅ | ✅ | ❌ |
-| hello_world     | ✅ | ✅ | ✅ |
-| http_server_example | ✅ | ❌ | ✅ |
-| i2c_example     | ✅ | ❌ | ❌ |
-| ledc_example    | ✅ | ❌ | ❌ |
-| read_priv       | ✅ | ✅ | ✅ |
-| spi_example     | ✅ | ❌ | ❌ |
-| system_info     | ✅ | ✅ | ✅ |
-| tcp_client      | ✅ | ❌ | ✅ |
-| tcp_server      | ✅ | ❌ | ✅ |
-| uart_example    | ✅ | ❌ | ❌ |
-| udp_client      | ✅ | ❌ | ✅ |
-| udp_server      | ✅ | ❌ | ✅ |
-| wifi            | ✅ | ❌ | ❌ |
+| Example Program | esp32 | stm32 | Pico | Pico W | generic_unix |
+|-----------------|-------|-------|----------------|------------------|--------------|
+| arepl_example   | ✅ | ❌ | ❌ | ✅ | ✅ |
+| blinky          | ✅ | ✅ | ✅ | ✅ | ❌ |
+| deep_sleep      | ✅ | ❌ | ❌ | ❌ | ❌ |
+| esp_nvs         | ✅ | ❌ | ❌ | ❌ | ❌ |
+| gpio_interrupt  | ✅ | ✅ | ✅ | ✅ | ❌ |
+| hello_world     | ✅ | ✅ | ✅ | ✅ | ✅ |
+| http_server_example | ✅ | ❌ |  | ✅ | ✅ |
+| i2c_example     | ✅ | ❌ | ❌ | ❌ | ❌ |
+| ledc_example    | ✅ | ❌ | ❌ | ❌ | ❌ |
+| read_priv       | ✅ | ✅ | ✅ | ✅ | ✅ |
+| spi_example     | ✅ | ❌ | ❌ | ❌ | ❌ |
+| system_info     | ✅ | ✅ | ✅ | ✅ | ✅ |
+| tcp_client      | ✅ | ❌ | ❌ | ✅ | ✅ |
```

Comment:
AFAIK, the internet examples still rely on gen_* and Pico OTP socket branch hasn't been merged yet